types/stripe-js/index.d.ts

```diff
@@ -259,6 +259,24 @@ declare module '@stripe/stripe-js' {
       data?: ConfirmAuBecsDebitSetupData
     ): Promise<{setupIntent?: SetupIntent; error?: StripeError}>;
 
+    /**
+     * Use `stripe.confirmBacsDebitSetup` in the [Bacs Direct Debit Payments](/docs/payments/payment-methods/bacs-debit) flow when the customer submits your payment form.
```

Comment:
This will need to be an absolute URL to work in users' editors:
```suggestion
     * Use `stripe.confirmBacsDebitSetup` in the [Bacs Direct Debit Payments](https://stripe.com/docs/payments/payment-methods/bacs-debit) flow when the customer submits your payment form.
```